code/modules/surgery/surgery.dm

```diff
@@ -116,6 +116,10 @@
 			prob_chance = allowed_tools[implement_type]
 		prob_chance *= get_location_modifier(target)
 
+		if(!istype(surgery.steps[surgery.status], /datum/surgery_step/robotics))//Repairing robotic limbs doesn't hurt
```

Comment:
So, this currently doesn't actually work as intended, for some reason. It's probably something really obvious I'm overlooking.